docs/DEMO_WALKTHROUGH.md

```
@@ -0,0 +1,514 @@
+- **Target Allocation:** Your desired distribution
+- **Drift Indicators:** Visual indicators showing how far each asset has drifted
+- Color coding: Green (within threshold), Yellow (approaching threshold), Red (exceeds threshold)
+
+### Performance Metrics
+- Total return ($ and %)
+- 24h change
+- 7-day change
+- Since inception performance
+
+### Asset Details Table
+| Asset | Current % | Target % | Drift | Value | Action Needed |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)